src/goto-symex/symex_target_equation.cpp

```diff
@@ -22,8 +20,7 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "goto_symex_state.h"
 
-symex_target_equationt::symex_target_equationt(
-  const namespacet &_ns):ns(_ns)
+symex_target_equationt::symex_target_equationt()
```

Comment:
How about removing this explicit definition and instead use `= default;` with the declaration?